src/types.rs

```diff
@@ -283,13 +287,15 @@ pub struct PeOptionalHeader64 {
 unsafe impl RefSafe for PeOptionalHeader64 {}
 
 #[repr(packed)]
+#[derive(Clone, Debug)]
```

Comment:
From a design point of view:
 - I personally think all types in this crate should be debug (as you probably guessed) as it's pretty useful for dumping information about PEs
 - Clone doesn't seem unreasonable - I just want it to copy whatever 4 bytes are in that position

From a practical point of view: RVA, FP, i32 and u32 are the only 4 types implementing `Size4Bytes`, and all of those satisfy both Clone and Debug - I can't realistically see any others being added.